src/Service/CloudFormation/src/Result/DescribeStacksOutput.php

```diff
@@ -200,23 +200,23 @@ private function populateResultStacks(\SimpleXMLElement $xml): array
                 'StackName' => (string) $item->StackName,
                 'ChangeSetId' => ($v = $item->ChangeSetId) ? (string) $v : null,
                 'Description' => ($v = $item->Description) ? (string) $v : null,
-                'Parameters' => !$item->Parameters ? [] : $this->populateResultParameters($item->Parameters),
```

Comment:
this is incompatible with the `stack::getParameter(): array` signature

Reply:
But have a look at the Stack constructor and in the Stack::getParameters()

This is safe

Reply:
Indeed. LGTM

Reply:
hmm.. after thinking about this.
I truly believe we should fix the constror of the ObjectValue instead. like we do for results:

https://github.com/async-aws/aws/blob/8b698e287967fa35a99246e99e69d45b6a47930f/src/Service/CognitoIdentityProvider/src/Result/AdminGetUserResponse.php#L144

It would have make sens to differentiate `null`/`[]` if the class exposed setters. 
But this is not the cases, and the getter converts null to [] anyway.

Working with array everywhere will be simpler IMHO